examples/lb/member/main.tf

```diff
@@ -0,0 +1,6 @@
+resource "vkcs_lb_member" "front_http" {
+  address       = "192.168.199.110"
+  protocol_port = 8080
+  pool_id       = vkcs_lb_pool.http.id
+  subnet_id     = vkcs_networking_subnet.app.id
```

Comment:
what is the reason for this argument?

Reply:
no sense for it. member looks on pool, pool on listener, listener on lb and lb on subnet.